### DISCUSSION TITLE
Can't open a single file from pCloud, or paste folders without errors, error log inside

### DISCUSSION THREAD

**Discussion by Hwmpunk** [iterate-ch/cyberduck#15484] [Q&A]
![Cyberduck error log](https://github.com/iterate-ch/cyberduck/assets/153105717/5eb626a8-efdf-473a-a87d-991c6ae36dce)

These files are not inside of a locked vault, just chilling in the main cloud folder. I can paste simple files into Cyberduck, but if i try to paste a folder with several contents i get an error, one after the other. They could be cannot create folder unexpected response 409 conflict, tons of upload failed unexpected response 403 forbidden, connection reset by peer socket write error, upload of directory failed unexpected response 500 internal server error multiple times, connection failed or outbound connection closed connection was rejected server may be down or network not properly configured, etc. 

I know pCloud isn't down or rejecting anything because in all other programs and operating systems it works just fine. And the folders I'm copying to paste in Cyberduck's encrypted vault I created in pCloud connection are coming from a nonencrypted Windows folder. I can copy paste these files anywhere else without any issues. Many times Cyberduck freezes on me, whether the entire program or just the transfers window.

I can add an encrypted vault to my pCloud but after setting a password on it, it never asks for it again.

**Comment by dkocher** [comment]
Do you refer to _pCloud Encryption_ or a _Cryptomator Vault_ supported in Cyberduck?

**Comment by dkocher** [comment]
I am currently unable to reproduce as the login with a test account to EU WebDAV endpoint `ewebdav.pcloud.com` always gives a `401` error valid credentials.

**Reply by dkocher** [reply]
> Thank you for contacting pCloud Support.
>
> We regret to inform you that WebDAV is no longer accessible for free plan users.

**Comment by WhatImListnTo** [comment]
Just as an FYI -- make sure you do not have TOTP authentication enabled for pCloud.  pCloud has zero support for TOTP with WebDAV.  They are aware of this.  The only fix for now is to disable TOTP for pCloud.

**Comment by Anja001** [comment]
Hello everyone, 

I have exactly the same problem. Unfortunately I couldn't find a correct solution here - *cry*.

@Hwmpunk: Were you able to solve the problem? If so, how?

@WhatImListnTo: I use Cyberduck in Windows. Where can I disable TOTP?

Thank you and best regards, 
Anja

**Reply by WhatImListnTo** [reply]
@Anja001 This is a pCloud setting.  Log into my.pcloud.com, click Settings, Security and disable Two-Factor Authentication to use WebDav.

**Reply by Anja001** [reply]
Ahh, now I have understood you.
But I have already disabled Two-Factor Authentication at my.pcloud.com.
So the problem should be somwehre else - unfortunately.
Any other ideas?

**Reply by WhatImListnTo** [reply]
I gave up on webdav finding it to perform poorly.  Sorry, but I cannot offer any suggestions.

**Reply by Anja001** [reply]
OK, thanks anyway.
